docs/infrastructure/cross-platform.md

```diff
@@ -17,7 +17,7 @@ The script will install .NET Core to `.dotnet` directory if it is not found on `
 
 ## Using the compiler
 
-After building, there will be a `csc` in the `artifacts/bin/csc/Debug/netcoreapp2.1` and `artifacts/bin/csc/Debug/net472` directories. Use the former to run on .NET Core and the latter on Mono.
+After building, there will be a `csc` in the `artifacts/bin/csc/Debug/netcoreapp3.1` and `artifacts/bin/csc/Debug/net472` directories. Use the former to run on .NET Core and the latter on Mono.
```

Comment:
❓ netcoreapp2.1 is LTS. Do we want to support both?

Reply:
The compiler no longer ships to the LTS SDK by default: only to `netcoreapp3.1` and forward. Any changes we `netcoreapp2.1` would be a servicing event but it's incredibly unlikely the compiler will meet the servicing bar. Instead we'd fix in current SDK and tell customers to use new SDK to build for `netcoreapp2.1` (this is not special to Roslyn but instead just how our SDK layer works)